src/main/java/com/epam/eco/commons/kafka/consumer/bootstrap/BootstrapConsumerExecutor.java

```diff
@@ -62,6 +63,12 @@ public void waitForBootstrap() throws InterruptedException {
         }
     }
 
+    public void waitForBootstrap(long timeout, TimeUnit timeUnit) throws InterruptedException {
```

Comment:
You are right, I misread doc and thought that await returns Exception in case of timeout. I have added a return value to waitForBootstrap to get the result of waiting